mmpose/models/losses/classification_loss.py

```diff
@@ -14,15 +14,17 @@ class BCELoss(nn.Module):
         use_target_weight (bool): Option to use weighted loss.
             Different joint types may have different target weights.
         loss_weight (float): Weight of the loss. Default: 1.0.
-        with_logits (bool): Whether to use BCEWithLogitsLoss. Default: False.
+        use_sigmoid (bool, optional): Whether the prediction uses sigmoid
+            of softmax. Defaults to False.
```

Comment:
```suggestion
        use_sigmoid (bool, optional): Whether the prediction uses sigmoid
            before output. Defaults to False.
```
Shall we use this kind of description?